site/docs/guides/operations/operate-omag-server-starter.md

```diff
@@ -1,18 +1,18 @@
 # Operate OMAG Server Starter
 
-This guide focuses on providing instructions how to operate server as stand-alone application process. This deployment choice gives you low level control of the application runtime and it requires direct interaction with your IT infrastructure.
+This guide focuses on providing instructions for running the server as standalone application process. This deployment option gives you low level over the application runtime and requires direct interaction with your IT infrastructure.
```

Comment:
unclear without the word "control", perhaps should be > gives you low level >of control< over

Reply:
Of course, you're right. I added the missing word and (hopefully) updated this PR.